### ISSUE TITLE
Zbay channel looks empty the first time (fresh account)

### ISSUE THREAD

**Issue by holmesworcester** [ZbayApp/zbay#752]
It seems like we need a loading indicator.

**Comment by holmesworcester** [comment]
What's the proposed solution here?

**Comment by EmiM** [comment]
The channel was empty not because messages were not loaded yet but because app was redirecting user to phantom zbay channel. For this particular problem I added one extra check and user will be redirected to real zbay channel only after this channel is pulled from waggle and set in contacts. https://github.com/ZbayApp/ZbayLite/pull/221

I propose that we would add loading indicators in other task.

**Comment by holmesworcester** [comment]
Loading indicators in a separate task is fine!
